application/controllers/admin/surveypermission.php

```diff
@@ -129,7 +129,7 @@ public function index($iSurveyID)
                         'uid' => $PermissionRow['uid']
                     ));
                     $deleteConfirmMessage = gT("Are you sure you want to delete this entry?");
-                    $surveysecurity .= "<a data-target='#confirmation-modal' data-toggle='modal' data-message='{$deleteConfirmMessage}' data-onclick='(function() {window.LS.sendPost(\"{$deleteUrl}\");})' type='submit' class='btn-xs btn btn-default'>
+                    $surveysecurity .= "<a data-target='#confirmation-modal' data-toggle='modal' data-message='{$deleteConfirmMessage}' data-onclick='(function() {" . convertGETtoPOST($deleteUrl) . "})' type='submit' class='btn-xs btn btn-default'>
```

Comment:
`convertGETtoPOST` is not deprecated ?

Reply:
> We can not copy function from Group and Question deletion ?
> Use same confirm box seems better (in my opinion).

Oh no : it's really an awfull code ...
https://github.com/LimeSurvey/LimeSurvey/blob/db67654da6e9ad311ff863273711301dfb96fd64/application/views/admin/survey/Question/questionbar_view.php#L128

Reply:
> `convertGETtoPOST` is not deprecated ?

No, still in use.